frontend/app/controllers/application.js

```diff
@@ -1,6 +1,23 @@
 import { inject as service } from '@ember/service';
 import Controller from '@ember/controller';
 
+
 export default Controller.extend({
-  session: service('session')
+  session: service('session'),
+  router: service(),
+
+  actions: {
+    transitionToProfile() {
+      let username = this.get('session.data.authenticated.full_name')
+      let people = this.get('store').findAll('person')
```

Comment:
Der Redirect zur Seite des Benutzer ist zweimal implementiert. Einmal hier und einmal in der login-route. Kann man das extrahieren/zentral aufrufen und so den Code nur an einer Stelle haben? Der einzige Unterschied ist, dass man in der login-route direkt `transitionTo` aufruft, während man hier noch erst den router holen muss.